envoy/router/router.h

```diff
@@ -383,7 +383,7 @@ class RetryState {
   /**
    * Determine whether a request should be retried based on the response headers.
    * @param response_headers supplies the response headers.
-   * @param original_request supplies the orignal request headers.
+   * @param original_request supplies the original request headers.
```

Comment:
Nit: unrelated, but OK.

Reply:
Yeah sorry this was causing my local push hooks to fail, not sure why this didn't fail CI. Didn't think it was worth splitting out.